crypto/pem/pem_lib.c

```diff
@@ -624,7 +624,7 @@ int PEM_write_bio(BIO *bp, const char *name, const char *header,
         (BIO_write(bp, "-----\n", 6) != 6))
         goto err;
 
-    i = strlen(header);
+    i = (header != NULL ? strlen(header) : 0);
```

Comment:
Please drop the parentheses around the ternary.

Reply:
I have dropped the parentheses.